src-docs/src/views/text_utilities/wrapping.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useContext } from 'react';
+
+import { ThemeContext } from '../../components/with_theme';
+
+import { EuiPanel, EuiCode } from '../../../../src';
+import { ThemeExample } from '../theme/_components/_theme_example';
+
+const maxWidth = 300;
+const longLink =
+  'http://www.hithereimalongurl.com/dave_will_just_ramble_on_in_a_long_sentence_like_this/?ok=cool';
+
+export default () => {
+  const themeContext = useContext(ThemeContext);
+  const currentLanguage = themeContext.themeLanguage;
+  const showSass = currentLanguage.includes('sass');
+
+  return (
+    <>
+      <ThemeExample
+        title={<code>.eui-textNoWrap</code>}
+        description={<p>Forces text not to wrap even in small containers.</p>}
+        example={
+          <EuiPanel
+            color="warning"
+            style={{ maxWidth }}
+            className="eui-textNoWrap"
+          >
+            This text will not to wrap but extend beyond the boundaries of the
+            yellow box.
+          </EuiPanel>
+        }
+        snippet={`<div className="eui-textNoWrap">
+  /* Your content */
+</div>`}
+      />
+
+      <ThemeExample
+        title={<code>.eui-textTruncate</code>}
+        description={
+          <>
+            <p>
+              Truncates text at 100% width of its parent and will display an
+              ellipsis.
+            </p>
+            <p>
+              <strong>Tip:</strong> When truncating text, it is recommended to
+              include the full text within an HTML <EuiCode>title</EuiCode>{' '}
+              attribute or by wrapping the element within an{' '}
+              <strong>EuiToolTip</strong>.
+            </p>
+          </>
+        }
+        example={
+          <EuiPanel
+            color="warning"
+            style={{ maxWidth }}
+            className="eui-textTruncate"
+          >
+            This text will not to wrap but truncate beyond the boundaries of the
+            yellow box.
+          </EuiPanel>
+        }
+        snippet={`<div
+  className="eui-textTruncate"
+  title={Your content}>
+  /* Your content */
+</div>`}
+      />
+      {/* Mixin */}
+      {!showSass ? (
+        <ThemeExample
+          title={<code>euiTextTruncate()</code>}
```

Comment:
Then I've been showing the possible parameters in the title

```suggestion
          title={<code>euiTextTruncate(maxWidth?)</code>}
```

Reply:
Ah I just saw this! Great idea!

Reply:
https://github.com/elastic/eui/pull/5854/commits/b0467f8d0c19afa676105d74b29d45fe457e78ba